redaxo/include/pages/mediapool.inc.php

```diff
@@ -130,7 +130,7 @@ function selectMedialist(filename)
     <?php
         if (substr($opener_input_field, 0, 14) == 'REX_MEDIALIST_') {
             $id = substr($opener_input_field, 14, strlen($opener_input_field));
-            echo 'var medialist = "REX_MEDIALIST_SELECT_' . $id . '";
+            echo 'var medialist = "REX_MEDIALIST_SELECT_' . htmlspecialchars($id) . '";
```

Comment:
an diesen stellen ist htmlspecialchars nicht das richtige (weil wir nicht direkt in html sind, sondern in einem script tag).

ich würde hier `(int)` casten.

Reply:
stimmt und an dieser stelle das nur ein int seien kann.